src/main.rs

```diff
@@ -130,8 +177,8 @@ enum Format {
     WithRelativeOffset(Expr, Box<Format>),
     /// Transform a decoded value with a function
     Map(Func, Box<Format>),
-    /// Conditional format
-    If(Expr, Box<Format>, Box<Format>),
```

Comment:
Although Match does subsume If but I think If might still be more succinct for matching on booleans! :sweat_smile:

Reply:
Used a helper function instead!